.github/workflows/packages.yml

```diff
@@ -198,31 +198,31 @@ jobs:
       - name: Install Grok Dependencies before package publish to Datagrok
         if: ${{ matrix.test == 'true' }}
         run: |
-          grok_deps="$(jq  -r '. | select( has("grokDependencies") == true ).grokDependencies | to_entries[] | .key' package.json)"
-          for dep in $grok_deps; do
-            current_dir=$(pwd);
-            cd ../;
-            dep_dir=$(grep -l "\"name\": \"$dep\"" */package.json | xargs -I {} dirname {});
-            cd $dep_dir;
-            npm install;
-            npm run build;
-            count=0
-            retries=5
-            until grok publish ${HOST}; do
-              exit=$?
-              wait=$((2 ** count))
-              count=$((count + 1))
-              if [ $count -lt "$retries" ]; then
-                echo "Retry $count/$retries exited $exit, retrying 'grok publish ${HOST}' in $wait seconds..."
-                sleep $wait
-                git pull --rebase
-              else
-                echo "Retry $count/$retries exited $exit, no more retries left for 'grok publish ${HOST}'."
-                exit $exit
-              fi
+          grok_deps="$(jq  -r '. | select( has("devDependencies") == true ).devDependencies | to_entries[] | .key' package.json | grep '@datagrok/')"
+          npm install
```

Comment:
We already ran instal earlier https://github.com/datagrok-ai/public/blob/dnahovskyi/GROK-12185-CI-install-packages-versions-from-grok-dependencies/.github/workflows/packages.yml#L166-L168